### DISCUSSION TITLE
Clean up excludes

### DISCUSSION THREAD

**Discussion by jon-rei** [vmware-tanzu/velero#7811] [Community support Q&A] [answered]
Hi all,

I want to exclude some specific PVCs that are really large from backups. These PVCs are already in our backups. I was wondering if it's possible to clean up the excluded resources from the S3 bucket without deleting the entire backups? I would really like to avoid deleting all the backups I have already done.

I'm currently using the latest Velero version with Kopia, CSI snapshot feature and AWS plugin.

**Comment by kaovilai** [comment]
Resources with the label `velero.io/exclude-from-backup=true` are not included in backup, even if it contains a matching selector label.
[src](https://velero.io/docs/v1.13/resource-filtering/#veleroioexclude-from-backuptrue:~:text=Resources%20with%20the%20label%20velero.io/exclude%2Dfrom%2Dbackup%3Dtrue%20are%20not%20included%20in%20backup%2C%20even%20if%20it%20contains%20a%20matching%20selector%20label.)

**Reply by kaovilai** [reply]
velero backups are not intended to be modified. I would create a new backup with above label applied to the PVCs

**Reply by jon-rei** [reply]
Thanks for your answer, I have already set this label.
For example, the PVCs I excluded are really large and I want to remove them from the S3 bucket to free up some space since they were accidentally included in the backup. My initial question was is there any logic or similar option to remove excluded resource from all backups that are already done and free up that space.
Since I did not find anything in the documentation about this, I would assume that I would have to delete my old backups to really free up some space.

**Reply by sseago** [reply, marked as answer]
@jon-rei That is correct. I would make sure you have newer backups without those large volumes, and then run `velero backup delete` to remove the old ones earlier than the delete-on-expiration action. There is no functionality in Velero for modifying existing backups.

**Reply by jon-rei** [reply]
Thanks for the information, I already thought this would be my only solution. Thanks for the confirmation.
